source/margarita/upgrading.rst

```diff
@@ -0,0 +1,557 @@
+
+#. Add ``myproject/load_env.py`` (same dir as root ``urls.py``):
+
+   .. code-block:: python
+
+      from os.path import dirname, join
+      import dotenv
+
+
+      def load_env():
+          "Get the path to the .env file and load it."
+          project_dir = dirname(dirname(__file__))
+          dotenv.read_dotenv(join(project_dir, '.env'))
+
+#. Modify ``myproject/celery.py``:
+
+   .. code-block:: diff
+
+      Modified   myproject/celery.py
+      diff --git a/myproject/celery.py b/myproject/celery.py
+      index d9a4e87..4f5a199 100644
+      --- a/myproject/celery.py
+      +++ b/myproject/celery.py
+      @@ -11,2 +11,5 @@ from celery import Celery
+
+      +from . import load_env
+      +load_env.load_env()
+      +
+
+#. Modify ``myproject/wsgi.py``:
+
+   .. code-block:: diff
+
+      Modified   myproject/wsgi.py
+      diff --git a/myproject/wsgi.py b/myproject/wsgi.py
+      index 69e0323..28cb28d 100644
+      --- a/myproject/wsgi.py
+      +++ b/myproject/wsgi.py
+      @@ -16,3 +16,5 @@ framework.
+       import os
+      +from . import load_env
+
+      +load_env.load_env()
+
+#. Modify ``manage.py``:
+
+   .. code-block:: diff
+
+      Modified   manage.py
+      diff --git a/manage.py b/manage.py
+      index cb48c9e..8bc2fce 100644
+      --- a/manage.py
+      +++ b/manage.py
+      @@ -4,2 +4,5 @@ import sys
+
+      +import dotenv
+      +dotenv.read_dotenv()
+      +
+       if __name__ == "__main__":
+
+#. Modify ``requirements/base.txt``:
+
+   .. code-block:: diff
+
+      Modified   requirements/base.txt
+      diff --git a/requirements/base.txt b/requirements/base.txt
+      index 2bb6ff2..ca74917 100644
+      --- a/requirements/base.txt
+      +++ b/requirements/base.txt
+      +
+      +django-dotenv==1.3.0
+
+#. Modify ``README.rst`` (and follow those instructions for your local setup):
+
+   .. code-block:: diff
+
+      Modified   README.rst
+      diff --git a/README.rst b/README.rst
+      index c45b564..8c456be 100644
+      --- a/README.rst
+      +++ b/README.rst
+      @@ -27,7 +27,9 @@ necessary requirements::
+
+      -Then create a local settings file and set your ``DJANGO_SETTINGS_MODULE`` to use it::
+      +Next, we'll set up our local environment variables. We use `django-dotenv
+      +<https://github.com/jpadilla/django-dotenv>`_ to help with this. It reads environment variables
+      +located in a file name ``.env`` in the top level directory of the project. The only variable we need
+      +to start is ``DJANGO_SETTINGS_MODULE``::
+
+      -    cp myproject/settings/local.example.py myproject/settings/local_dev.py
+      -    echo "export DJANGO_SETTINGS_MODULE=myproject.settings.local_dev" >> $VIRTUAL_ENV/bin/postactivate
+      -    echo "unset DJANGO_SETTINGS_MODULE" >> $VIRTUAL_ENV/bin/postdeactivate
+      +    (myproject)$ cp myproject/settings/local.example.py myproject/settings/local.py
+      +    (myproject)$ echo "DJANGO_SETTINGS_MODULE=myproject.settings.local" > .env
+
+
```

Comment:
I mention that below, in step 4 under 'Miscellaneous Work'. Would you prefer it here? Or maybe in both places?